addons/advanced_ballistics/functions/fnc_handleFired.sqf

```diff
@@ -103,7 +103,7 @@ if (GVAR(bulletTraceEnabled) && cameraView == "GUNNER") then {
 
 private _stabilityFactor = 1.5;
 if (_caliber > 0 && _bulletLength > 0 && _bulletMass > 0 && _barrelTwist > 0) then {
-    if (isNull _temperature) then {
+    if (_temperature isEqualTo scriptNull) then {//isNull doesn't work with Numbers
```

Comment:
Elaborate? SCRIPT type is different from NUMBER. Although `isNull` does indeed not work with NUMBER, it should work with SCRIPT just fine.
https://community.bistudio.com/wiki/isNull
Therefore, `_temperature isEqualTo scriptNull` should be `isNull temperature`.

Reply:
I expect to have isNull check if variable contains a null type. And return false if it's not null. But instead of returning false on a number it just errors.
_temperature is by default set to scriptNull. So here I just check if it's still default(undefined)